library/src/main/java/io/github/wax911/library/annotation/processor/GraphProcessor.kt

```diff
@@ -66,18 +71,35 @@ class GraphProcessor private constructor(assetManager: AssetManager?) {
                 paths?.also {
                     for (item in it) {
                         val absolute = "$path/$item"
-                        if (!item.endsWith(defaultExtension))
-                            createGraphQLMap(absolute, this)
-                        else
+                        if (item.endsWith(defaultExtension))
                             _graphFiles[item] = getFileContents(open(absolute))
+                        else
+                            createGraphQLMap(absolute, this)
                     }
                 }
             }
+
+            patchQueries()
```

Comment:
I really wanted to apply the patching within the file iteration above. The problem is, all files need to be discovered first. That way the full set of available fragments is available for patching.